504.xml

```diff
@@ -327,22 +339,24 @@ LwM2M Client.]]></Description>
 			<Item ID="22">
 				<Name>Integrated Circuit Card Identifier (ICCID)</Name>
 				<Operations>R</Operations>
-				<MultipleInstances>Single</MultipleInstances>
-				<Mandatory>Optional</Mandatory>
-				<Type>Integer</Type>
+				<MultipleInstances>Multiple</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>String</Type>
 				<RangeEnumeration></RangeEnumeration>
 				<Units></Units>
-				<Description><![CDATA[This EF provides a unique identification number for the UICC/Smart Cards. Please refer ETSI TS 102.22.1]]></Description>
+				<Description><![CDATA[This resource provides the unique identification number for each installed profile of the current SIM in case of RSP support or
+the unique identification number of the current SIM in case of current SIM being a UICC with no RSP support. Please refer ETSI TS 102.22.1. In case of multiple installed profiles,
+the currently active profile is the first instance.]]></Description>
 			</Item>
 			<Item ID="23">
 				<Name>eUICC ID</Name>
 				<Operations>R</Operations>
 				<MultipleInstances>Single</MultipleInstances>
 				<Mandatory>Optional</Mandatory>
-				<Type>Integer</Type>
+				<Type>String</Type>
 				<RangeEnumeration></RangeEnumeration>
 				<Units></Units>
-				<Description><![CDATA[eUICC-ID as defined in SGP.02 spec]]></Description>
+				<Description><![CDATA[eUICC-ID (a.k.a. EID), see GSMA SGP.02 and GSMA SGP.29 for definitions]]></Description>
 			</Item>
 			<Item ID="24">
```

Comment:
This is the profile type in consumer variant given on ES2+ I/F when indicating to SM-DP+ what type of profile to create. How is this information is encoded in the profile package such that it can be provided from the LwM2M client to the LwM2M server? Can you please provide information on the foreseen usage of this resource?
There is a reference [4] in the description of the resource. What is it referring to?